earth_enterprise/BUILD_RHEL_Centos.md

```diff
@@ -0,0 +1,121 @@
+# GEE Build Setup for CentOS, RHEL 6 and RHEL 7     
+
+
+## Install git  
+    
+Install the system default version:
+    
+```
+sudo yum install git
+```
+    
+Or install the latest version from the IUS repo (recommended) [ [More Info] ](https://ius.io/GettingStarted/):
+```
+sudo yum install -y wget
+cd /tmp
+wget https://rhel7.iuscommunity.org/ius-release.rpm
+sudo yum install ius-release.rpm
+sudo yum install git2u-all
+```
+        
+        
+## Install git lfs
+   
+Execute the following commands
+    
+```
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash
+sudo yum install git-lfs
+```
+
+
+
+## Enable additional repos
+Different yum repositoies need to be enabled for different distributions to install several required development tools.  
+
+### CentOS 7 
+```
+sudo yum install epel-release
+```
+
+### RHEL 7  
+
+```
+sudo subscription-manager repos --enable=rhel-7-server-extras-rpms
+sudo subscription-manager repos --enable=rhel-7-server-optional-rpms
+sudo subscription-manager repos --enable=rhel-7-server-optional-source-rpms
+sudo yum install epel-release
+```    
+
+### RHEL 6
+```
+sudo subscription-manager repos --enable=rhel-x86_64-workstation-dts-2
+sudo subscription-manager repos --enable=rhel-6-server-optional-rpms
+sudo subscription-manager repos --enable=rhel-6-server-optional-source-rpms
+```
+       
+## GCC 4.8
+
+### CentOS 7 and RHEL 7
+
+To obtain GCC 4.8 on RHEL6, you must enable the "Development Tools" repository in the RHSM: 
```

Comment:
I wouldn't call this enabling a repository. The following command is installing a group of packages, using the Yum repositories that are already set up.